frontend/src/components/pages/CampCreation/CampDetails/index.tsx

```diff
@@ -1,49 +1,520 @@
+            borderWidth={!startTime && showErrors?"2px":"1px"}   
+            onChange={handleStartTime}/> 
+          {errorText(startTime,"You must specify a time.")}
+        </Box>
+        
+        <Text paddingTop="45px"> to </Text>
+        
+        <Box width="160px">
+          <Text textStyle="buttonSemiBold">
+            End Time{" "}
+            <Text textStyle="buttonSemiBold" display="inline" color="red">*</Text>
+          </Text>
+        
+          <Input type="time" placeholder='XX:XX' width="160px" height="52px" marginTop="8px"
+            value={endTime}
+            borderColor={!endTime && showErrors?"red":"gray.200"}
+            borderWidth={!endTime && showErrors?"2px":"1px"}  
+            onChange={handleEndTime}/>
+          {errorText(endTime,"You must specify a time.")}
+        </Box>
+      </HStack>
+
+      <HStack alignItems="start" spacing={8} marginTop="24px" marginBottom="24px">
+        <Box>
+          <Text textStyle="buttonSemiBold">
+            Age Range{" "} 
+            <Text textStyle="buttonSemiBold" display="inline" color="red">*</Text>
+          </Text>
+          <HStack alignItems="start" spacing={4} marginTop="8px">
+            <Box width="100px">
+              <Input type="number" width="100px" height="52px" maxLength={2}
+                value={ageLower}
+                borderColor={!ageLower && showErrors?"red":"gray.200"}
+                borderWidth={!ageLower && showErrors?"2px":"1px"}onChange={handleAgeLower}/>
+              {errorText(ageLower,"You must enter an age.")}
+            </Box>
+            <Text paddingTop="14px"> to </Text>
+            <Box width="100px">
+              <Input type="number" width="100px" height="52px" maxLength={2}
+                value={ageUpper}
+                borderColor={!ageUpper && showErrors?"red":"gray.200"}
+                borderWidth={!ageUpper && showErrors?"2px":"1px"}onChange={handleAgeUpper}/>
+              {errorText(ageUpper,"You must enter an age.")} 
+            </Box>
+          </HStack>
+        </Box>
+
+        <Box>
+            <Text textStyle="buttonSemiBold">
+              Camp Capacity {" "}
+              <Text textStyle="buttonSemiBold" display="inline" color="red">*</Text>
+            </Text>
+            <Box width="200px">
+              <Input type="number" width="200px" height="52px" marginTop="8px"
+                value={campCapacity}
+                borderColor={!campCapacity && showErrors?"red":"gray.200"}
+                borderWidth={!campCapacity && showErrors?"2px":"1px"}onChange={handleCampCapacity}/> 
+              {errorText(campCapacity,"You must enter a number.")} 
+            </Box>
+        </Box>
+      </HStack>
+      
+      <Checkbox marginTop="8px" onChange={toggleEDLP}>
+        <Text textStyle="buttonSemiBold">
+          Camp offers early drop-off and late pick-up
+        </Text>
+      </Checkbox>
+      
+
+      {offersEDLP?(
```

Comment:
I think you can use `offersEDLP && ()` so you don't have the render the empty component with the ternary operator